### ISSUE TITLE
machine coordinates!!!

### ISSUE THREAD

**Issue by gabrail-110** [Denvi/Candle#388]
The machine coordinates do not exceed 999,000 as you can see in the picture
![1+](https://user-images.githubusercontent.com/65731573/82592378-106aa880-9bb6-11ea-923a-85f4d033cc8b.png)

**Comment by ASHIT-AXAR** [comment]
true. also it's working correctly, but doesn't show in the coordinate display.

**Comment by Denvi** [comment]
Fixed in [1.2.9b](https://github.com/Denvi/Candle/releases/download/v1.2b/Candle_1.2.9b.zip).

**Comment by gabrail-110** [comment]
Hi, thank you, but there has been no change in the software
![Snap 2020-06-09 at 00 14 05](https://user-images.githubusercontent.com/65731573/84075104-d50b0f00-a9e8-11ea-89a0-03f4e2ec4cea.png)
![Snap 2020-06-09 at 00 14 53](https://user-images.githubusercontent.com/65731573/84075115-d805ff80-a9e8-11ea-8c27-4d45d7a403e3.png)

**Comment by Denvi** [comment]
Ok, how about [1.2.9.1b](https://github.com/Denvi/Candle/releases/download/v1.2b/Candle_1.2.9.1b.zip)

**Comment by gabrail-110** [comment]
It works properly

**Comment by gabrail-110** [comment]
The software works great in terms of communication with the microcontroller and displays visual movement, since I'm just a user and I'm not a programmer, but I'm interested in learning programming.
